tensorflow/tools/pip_package/build_pip_package.sh

```diff
@@ -17,6 +17,10 @@
 
 set -e
 
+function real_path() {
```

Comment:
When I try that on macOS, I see this:
```
gunan$ readlink -f .
readlink: illegal option -- f
usage: readlink [-n] [file ...]
```

readlink -f seems to be OK with GNU coreutils, but it does not seem to work with BSD coreutils.